eng/Versions.props

```diff
@@ -23,6 +23,7 @@
     <MicrosoftCodeAnalysisCSharpVersion>4.6.0</MicrosoftCodeAnalysisCSharpVersion>
     <MicrosoftCodeAnalysisPublicApiAnalyzersVersion>3.3.4</MicrosoftCodeAnalysisPublicApiAnalyzersVersion>
     <MicrosoftCodeAnalysisBannedApiAnalyzersVersion>$(MicrosoftCodeAnalysisPublicApiAnalyzersVersion)</MicrosoftCodeAnalysisBannedApiAnalyzersVersion>
+    <NuGetPackagingCore>6.8.0</NuGetPackagingCore>
```

Comment:
Please move this deps to the test deps section (it's only used for testing). Also please rename the property to end with `Version`.

Reply:
removed